src/docker-extract.c

```diff
@@ -244,6 +259,7 @@ int extract_tar(const char *tarfile, const char *rootfs_dir) {
     const char *pathname;
     int pathtype;
 
+    printf("EXTRACT TAR NAME: %s\n", tarfile);
```

Comment:
Anything that is a log message needs to be written out using the `singularity_message` function, so it can be silenced/shown by the set debug level.

Reply:
Probably this actually could be removed - since the outer wrappers for building etc. will log the tar which is going to be extracted.

Reply:
So sorry, that was for my consumption only, I can't believe I left it. :)